### PR TITLE
7z: Added output path example.

### PR DESCRIPTION
**Pull request by shahril96**
Added output path example into 7z man-page.

----

- [ ] The page (if new), does not already exist in the repo.

- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/7z.md [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 `7z x {{archived}}`
 
+- Extract an archive with user-defined output path
```
Comment (pxgamer) [inline pages/common/7z.md]: Just need to add a colon to the end of this description.

File: pages/common/7z.md [hunk 2]
```diff
@@ -10,6 +10,10 @@
...
+
+`7z x {{archived}} -o{{output/path}}`
```
Comment (jsonbruce) [inline pages/common/7z.md]: I suggest let's use `path/to/output` to keep consistency.

Comment (pxgamer) [reply pages/common/7z.md]: Good point 👍

**Comment by shahril96** [comment]
Hi @pxgamer and @jsonbruce, those have been fixed.
Thanks for your comments. :)

**Comment by pxgamer** [comment]
Hi @shahril96, all looks good to me. Just need to sign the CLA and all should be good. 👍

**Comment by shahril96** [comment]
I've signed the CLA, so everything should be ok now. Thanks, everybody.

**Comment by pxgamer** [comment]
Just wating for Travis. There's a bit of a backlog.

File: pages/common/7z.md [hunk 3]
```diff
@@ -10,6 +10,10 @@
 
 `7z x {{archived}}`
 
+- Extract an archive with user-defined output path:
+
+`7z x {{archived}} -o{{path/to/output}}`
```
Comment (agnivade) [inline pages/common/7z.md]: Shouldn't there be a space between "o" and "{" ?

Comment (pxgamer) [reply pages/common/7z.md]: I thought that too at first, but it looks like it doesn't have a space according to the docs/man page.

Comment (shahril96) [reply pages/common/7z.md]: Nope, it doesn't have space between -o and {}.

Reference: http://7zip.bugaco.com/7zip/MANUAL/switches/output_dir.htm

Comment (agnivade) [reply pages/common/7z.md]: Cool.

**Pull request merged**